src/components/toast/toast.tsx

```diff
@@ -38,9 +38,9 @@ export const InternalToast: React.FC<ToastProps> = p => {
     if (icon === null || icon === undefined) return null
     switch (icon) {
       case 'success':
-        return <CheckOutline />
+        return <CheckOutline className={`${classPrefix}-success`} />
```

Comment:
这里还是加个 icon 的前缀吧～

```suggestion
        return <CheckOutline className={`${classPrefix}-icon-success`} />
```

下同

Reply:
ok